cibuildwheel/util.py

```diff
@@ -1,5 +1,11 @@
 from fnmatch import fnmatch
 
+filter_wheels_cmd = '''\
+import sys, glob, itertools, pip; \
+sys.stdout.write(\
+' '.join(f for f in itertools.chain(*map(glob.iglob, sys.argv[1:])) \
+if pip.wheel.Wheel(f).supported()))\
+'''
```

Comment:
Sorry, lack of commenting. I can add some.
This is a short python program that takes as arguments a list of wheels and prints out only those wheels that are supported on the executor's platform. I put it here since all platforms use it and there's no need to have modify it per-platform.

But if you're asking why...hopefully my [previous comment](https://github.com/joerick/cibuildwheel/pull/18#issuecomment-313952707) makes it clear.

> Also I think this is an undocumented Pip API... Might change in the future.

I will try and verify whether that's the case. I tend to assume anything not starting with an underscore is fairly stable but this may not be true in pip world.